audit-ci.jsonc

```diff
@@ -4,11 +4,9 @@
   // Only use one of ["low": true, "moderate": true, "high": true, "critical": true]
   "moderate": true,
   "allowlist": [
-    // @mojaloop/sdk-standard-components>jsonwebtoken
-    "GHSA-hjrf-2m68-5959",
-    "GHSA-qwph-4952-7xr6",
-    "GHSA-8cf7-32gw-wr33",
     // request-promise-native>request-promise-core>request
-    "GHSA-p8p7-x288-28g6"
+    "GHSA-p8p7-x288-28g6",
+    "GHSA-c2qf-rxjj-qqgw",
+     "GHSA-72xf-g2v4-qvf3"
```

Comment:
Can you add some comments to indicate why these are in exception list?

Reply:
added a comment and created an issue to change library since the request library is deprecated: https://github.com/mojaloop/project/issues/3545